lib/discordgo/structs.go

```diff
@@ -1421,6 +1421,8 @@ type CreateApplicationCommandRequest struct {
 	Options           []*ApplicationCommandOption `json:"options"`                      // the parameters for the command
 	DefaultPermission *bool                       `json:"default_permission,omitempty"` // (default true)	whether the command is enabled by default when the app is added to a guild
 	NSFW              bool                        `json:"nsfw,omitempty"`               // marks a command as age-restricted
+	Integration_Types []int                       `json:"integration_types"`            // Defines the integration types a command supports https://discord.com/developers/docs/resources/application#application-object-application-integration-types
```

Comment:
Go linters have this saying: "don't use underscores in go names"
https://go.dev/doc/effective_go#mixed-caps

same for for variable `integration_types` you have in slashcommands.go

Reply:
Will remove when home